cmake/modules/FindFastCGI.cmake

```diff
@@ -7,41 +7,28 @@
 # Find the FastCGI includes and library
 #
 #  FASTCGI_INCLUDE_DIR - where to find fcgiapp.h
-#  FASTCGI_LIBRARY     - librart when using MySQL.
-#  FASTCGI_FOUND       - True if FASTCGI found.
+#  FASTCGI_LIBRARY     - library when using FastCGI.
+#  FASTCGI_FOUND       - true if FASTCGI found.
 
-if(FASTCGI_INCLUDE_DIR OR FASTCGI_)
-  # Already in cache, be silent
-  SET(FASTCGI_FIND_QUIETLY TRUE)
+if(NOT FASTCGI_INCLUDE_DIR)
+   find_path(FASTCGI_INCLUDE_DIR NAME fcgiapp.h PATH_SUFFIXES include)
```

Comment:
This `if` is not necessary, as if the variable is already in the cache, the command will not do anything.